page-settings/home.js

```diff
@@ -61,13 +61,16 @@ export const PAGES = [
     title: "Real Time Traffic Incidents",
     description: "View active traffic incidents reported to 911",
     href: "https://data.austintexas.gov/stories/s/Austin-Travis-County-Traffic-Report-Page/9qfg-4swh/",
-    img: { src: "/assets/traffic-incidents.jpg", alt: "Traffic incidents image" },
+    img: {
+      src: "/assets/traffic-incidents.jpg",
+      alt: "Traffic incidents image",
+    },
     external: true,
   },
   {
     title: "Open Data",
     description: "Browse and download the data that powers our operations",
-    href: "https://data.austintexas.gov/browse?Ownership_Department-name=Austin+Transportation+%26+Public+Works&limitTo=datasets",
+    href: "https://data.austintexas.gov/browse?Ownership_Department-name=Austin+Transportation+and+Public+Works&limitTo=datasets",
```

Comment:
h/t @chiaberry - thanks for catching this rogue bug during testing!

Reply:
thanks for updating, Frank, and thanks for catching, Chia! i think the "Austin" part still needs to be removed to match up with department name on the open data portal to get this link to filter to our datasets. <img width="236" alt="Screenshot 2025-06-09 at 5 29 47 PM" src="https://github.com/user-attachments/assets/919f3d5e-b09a-4382-8135-90a584069b0e" />

Reply:
🙏🙏🙏 Thank you for testing that. https://github.com/cityofaustin/atd-data-and-performance/pull/423/commits/0fe252e00dfeb5a27c3e6c1f54b45e7ebe7d79f3. This worked for me locally.